src/wpool_queue_manager.erl

```diff
@@ -190,7 +196,11 @@ handle_cast({cast_to_available_worker, Cast}, State) ->
       {Worker, NewWorkers} = gb_sets:take_smallest(Workers),
       ok = wpool_process:cast(Worker, Cast),
       {noreply, State#state{workers = NewWorkers}}
-  end.
+  end;
+handle_cast({cast_to_worker, Cast, Worker}, State) ->
```

Comment:
If you're not using the queue, there is no reason for this indirection of messages, you can directly use `wpool_process:cast/2` from the outside.